server/account/src/index.ts

```diff
@@ -234,13 +234,13 @@ function toAccountInfo (account: Account): AccountInfo {
 async function getAccountInfo (ctx: MeasureContext, db: Db, email: string, password: string): Promise<AccountInfo> {
   const account = await getAccount(db, email)
   if (account === null) {
-    throw new PlatformError(new Status(Severity.ERROR, platform.status.AccountNotFound, { account: email }))
```

Comment:
I think we still need log states, InvalidPassword into context logger and pass them into analytics, so we could understand and track a problems.